### PR TITLE
allow doctrine/annotations 2

### LINKED ISSUE #612: Dependecies upgrade: doctrine/annotations
Can you upgrade the dependencies doctrine/annotations to let's use the last version with your dependencies please.

### Steps to reproduce
Here my require
`
"require": {
        "php": "^8.0.3",
        "ext-ctype": "*",
        "ext-iconv": "*",
        "doctrine/annotations": "^2.0",
        "doctrine/doctrine-bundle": "^2.8",
        "doctrine/doctrine-migrations-bundle": "^3.2",
        "doctrine/orm": "^2.14",
        "phpdocumentor/reflection-docblock": "^5.3",
        "phpstan/phpdoc-parser": "^1.16",
        "symfony/console": "5.4.*",
        "symfony/dotenv": "5.4.*",
        "symfony/flex": "^1.17|^2",
        "symfony/form": "5.4.*",
        "symfony/framework-bundle": "5.4.*",
        "symfony/property-access": "5.4.*",
        "symfony/property-info": "5.4.*",
        "symfony/runtime": "5.4.*",
        "symfony/security-bundle": "5.4.*",
        "symfony/serializer": "5.4.*",
        "symfony/translation": "5.4.*",
        "symfony/validator": "5.4.*",
        "symfony/web-profiler-bundle": "5.4.*",
        "symfony/yaml": "5.4.*"
    }
`
And the error message when i want install your library
`
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires liip/functional-test-bundle ^4.7 -> satisfiable by liip/functional-test-bundle[4.7.0].
    - liip/functional-test-bundle 4.7.0 requires doctrine/annotations ^1.3 -> found doctrine/annotations[v1.3.0, ..., 1.14.3] but it conflicts with your root composer.json require (^2.0).
`

### PR DESCRIPTION
**Pull request by alexislefebvre**
Fixes #612

### CODE REVIEW

**Comment by alexislefebvre** [comment]
Should this dependency be not mandatory?

**Comment by Nirator78** [comment]
In the llip library you are using it only in the class QueryCounter

**Comment by alexislefebvre** [comment]
Yes, the bundle may not require `doctrine/annotations` if users don't use the `QueryCounter`.

**Comment by Nirator78** [comment]
So what we can do ? Delete the `doctrine/annotations` or resolve the test to allow upgrade the library ?

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -65,6 +65,14 @@ jobs:
           symfony/validator=${{ matrix.symfony-version }}
           symfony/yaml=${{ matrix.symfony-version }}
 
+      - name: Require Annotations v1 to require for PHP 7
+        if: startsWith(matrix.php-version, '7.')
+        run: composer require --no-update "doctrine/annotations:^1.8.0"
+
+      - name: Require Annotations v1/v2 to require for PHP 8+
+        if: startsWith(matrix.php-version, '7.') != true
```
Comment (alexislefebvre) [inline .github/workflows/tests.yml:73]: Reference: https://github.com/orgs/community/discussions/25017#discussioncomment-3246240

**Comment by alexislefebvre** [comment]
I had to force `doctrine/annotations` v1 on PHP 7.* otherwise the CI failed, we'll check this later.

**Pull request merged**